### PR TITLE
Bugfix/supesc 600 guest cart is not automatically assigned to an authenticated user

### PR DESCRIPTION
**Pull request by Zingeon**
## PR Description

Ticket: https://spryker.atlassian.net/browse/SUPESC-600

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by VadymSachenko — COMMENTED** [review]
The doc has been reviewed.

**Review by dmytro-sokolovskyi — COMMENTED** [review]
Please, add the link to the ticket in PR Description.

**Review by dmytro-sokolovskyi — APPROVED** [review]

File: _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-cart-glue-api.md [hunk 1]
```diff
@@ -180,6 +180,25 @@ class CartsRestApiConfig extends SprykerCartsRestApiConfig
 }
 ```
 
+### Configure quote creation
+
+If you want to create a cart for the newly authenticated customer while merging guest cart with customer cart, you can enable it by adjusting the configuration constant:
```
Comment (VadymSachenko) [inline _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-cart-glue-api.md:185]: @Zingeon  In the phrase "you can enable it", what do you mean by "it"?

Comment (Zingeon) [reply _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-cart-glue-api.md:185]: @VadymSachenko the following rephrasing sounds good? `You can enable creating a cart for the newly authenticated customer while merging guest cart with customer cart by adjusting the configuration constant:`

Comment (VadymSachenko) [inline _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-cart-glue-api.md:185]:
```suggestion
You can enable the creation of a cart for the newly authenticated customer while merging the guest cart with the customer cart by adjusting the configuration constant:
```

File: docs/scos/dev/feature-integration-guides/202108.0/glue-api/glue-api-cart-feature-integration.md [hunk 2]
```diff
@@ -195,6 +195,25 @@ class CartsRestApiConfig extends SprykerCartsRestApiConfig
 }
 ```
 
+### Configure quote creation
+
+If you want to create a cart for the newly authenticated customer while merging guest cart with customer cart, you can enable it by adjusting the configuration constant:
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202108.0/glue-api/glue-api-cart-feature-integration.md:200]:
```suggestion
You can enable the creation of a cart for the newly authenticated customer while merging the guest cart with the customer cart by adjusting the configuration constant:
```

**Review by VadymSachenko — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**